config/webpacker.yml

```diff
@@ -6,7 +6,7 @@ default: &default
   public_root_path: public
   public_output_path: packs
   cache_path: tmp/cache/webpacker
-  check_yarn_integrity: true
+  check_yarn_integrity: false
```

Comment:
can we revert this, so the "default" is true on this?

Reply:
In: d86bb5e